app/views/profiles/show.html.erb

```diff
@@ -20,6 +20,10 @@
           <p><em><%= trip.crag.name %></em></p>
           <p><%= trip.start_date %> ⇨ <%= trip.end_date %></p>
         </div>
+         <% if @profile.user == current_user %>
```

Comment:
this is always the case, right?

Reply:
No, it's not. If I visit your profile, I am the current user, but @profile.user is you :)